next.config.js

```diff
@@ -24,6 +24,11 @@ const moduleExports = {
         source: '/book',
         destination: '/docs/learn',
         permanent: true
+      },
+      {
+        source: '/api/auth/callback/discord',
+        destination: '/discord/success',
+        permanent: true
```

Comment:
I think we need to update the discord application uri redirect setting and not try to have next do the redirect

Reply:
I tried doing res.redirect from the /api/auth/callback/discord handler, according to the docs:

https://nextjs.org/docs/api-routes/response-helpers

But I kept getting errors. This was my workaround, heh.

<img width="709" alt="res redirect" src="https://user-images.githubusercontent.com/16906103/134760111-8f189ac9-cbed-4dd8-a3f3-0cccd1bf2c4c.png">
.

Reply:
That would still be a next redirect solution, I'm talking about setting the discord app to just send us to the right page either a success page or back to curriculum to the curriculum page.

Then we could merge the functionality of the discord/api/auth page into that page with a useEffect on the `code` query string

https://discord.com/developers/applications/ :our app id:/oauth2
![discordRedirect](https://user-images.githubusercontent.com/17365077/134773202-d252a81f-9528-445d-a58d-0c7eeb63d011.png)

Reply:
> I tried doing res.redirect from the /api/auth/callback/discord handler, according to the docs:
> 
> https://nextjs.org/docs/api-routes/response-helpers
> 
> But I kept getting errors. This was my workaround, heh.
> 
> <img alt="res redirect" width="709" src="https://user-images.githubusercontent.com/16906103/134760111-8f189ac9-cbed-4dd8-a3f3-0cccd1bf2c4c.png">
> 
> .

Though I would think the res.redirect("/discord/success") should work